mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -1131,28 +1131,33 @@ void printLoopControl(OpAsmPrinter &p, Operation *op, Region &region,
 }
 
 //===----------------------------------------------------------------------===//
-// Verifier for Simd construct [2.9.3.1]
+// Verifier for Simd constructs [2.9.3.1]
```

Comment:
Nit: 2.9.3.1 refers to `omp simd` construct. Could you add referral to `omp do simd` as well? It's 2.9.3.2